README.md

```diff
@@ -1,2 +1,37 @@
 # ACM-ICPC Algorithms
-Algorithms used in Competitive Programming
+
+What is **ACM-ICPC**?
+
+ACM International Collegiate Programming Contest (abbreviated as ACM-ICPC or ICPC) is an annual multi-tiered competitive programming competition among the universities of the world
+
+For more information visit : 
+
+>https://icpc.baylor.edu/
+
+>https://en.wikipedia.org/wiki/ACM_International_Collegiate_Programming_Contest
+
+What is **ACM-ICPC Algorithms**?
+
+ACM-ICPC Algorithms is your personal offline collection of every algorithm and data structure used to solve the questions of this worldwide recognized Olympiad.This provides solutions in a various languages spanning C, C++, Java, Python and others.
+
+
+**Table of Contents**
+
+
+* Breadth First Traversal
+* Backtracking
+* DFS
+* Bit manipulation
+* Computational geometry
+* Data structures
+* Divide conquer
+* Dynamic programming
+* Graph algorithms 
+* Greedy algorithms
+* Mathematical algorithms
+* Randomized algorithms
```

Comment:
Randomized algorithms are not here

Reply:
Ohh, i haven't noticed

Reply:
just a sec i will add these algos

Reply:
Also links would be nice

Reply:
i will create it as issue just assign it to me.